.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+version: 2.1
+jobs:
+  setup:
+    docker:
+      - image: circleci/ruby:3.0.1-node
+        environment:
+          NODE_ENV: test
+          RAILS_ENV: test
+          POSTGRES_HOST: localhost
```

Comment:
Do we need this Postgres config here

Reply:
Yes, without it   "run: bundle exec rails webpacker:compile" fails with key_error. Was able to resolve it after adding config there